src/SamplesApp/UITests.Shared/Windows_UI_Input/VisualStatesTests/TextBoxPointerStates.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<Page
+	x:Class="UITests.Shared.Windows_UI_Input.VisualStatesTests.TextBoxPointerStates"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:UITests.Shared.Windows_UI_Input.VisualStatesTests"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+	
+	<Grid>
```

Comment:
Normal that this page is empty ?

Reply:
Not really ... will remove/complete it (in a future commit)